README.md

```diff
@@ -0,0 +1,22 @@
+# Overture Iobio Components
+
+- A set of reusable React components and command line tools for integrating the visualization and analysis software originally created by https://iobio.io/
+
+- This project is part of the [Overture](https://www.overture.bio/) research software ecosystem. See our [related products](#related-products) for more information on how Overture is helping organize data and enable discovery.
```

Comment:
what `#related-products` are you referring to? maybe the link is meant to go somewhere in overture.bio? or is it WIP in this readme?

Reply:
Sorry lazy copy from Stage. I'm going to update to use the template: https://main--overturedev.netlify.app/docs/standards/software/readme/

Reply:
This is fixed, added an anchor to the related header